docs/notebooks/utils/.ipynb_checkpoints/downloading_brains-checkpoint.ipynb

```diff
@@ -0,0 +1,145 @@
+{
```

Comment:
Is this file supposed to be here? What is it for?

Reply:
I think not. To not include these files in your commits, add `**-checkpoint.ipynb` to the `.gitignore` file. You may have to clear your git cache afterwards.